test/test_other.py

```diff
@@ -7118,6 +7118,7 @@ def test_realpath_2(self):
 Resolved: "/" => "/"
 ''', self.run_js('a.out.js'))
 
+  @with_env_modify({'EMCC_LOGGING': '0'})  # this test assumes no emcc output
```

Comment:
This feels a bit odd to me. It disables logging, but the test is verifying that there is no problem that causes logging. So it is really a test for the flag disabling logging, but not for the problems that can cause logging existing or not.

In practice most of those warnings are from clang, which `EMCC_LOGGING` doesn't handle. So I'm not too worried here in practice, I guess.

Reply:
`EMCC_LOGGING` only effect logging.. which is completely separate to the warnings system.  `EMCC_LOGGING` does not effect warnings.

Reply:
I see. That's less worrying to me then.

Reply:
Perhaps its a little confusing... that emscripten uses both python's logging module and also its own "diagnostics.py" module?